### DISCUSSION TITLE
indi-allsky/gunicorn hanging

### DISCUSSION THREAD

**Discussion by jshall77** [aaronwmorris/indi-allsky#956] [Q&A]
Hello,
I am relatively new to indi-allsky, however, i used the regular "Allsky" for quite a while before finding this variant.  I love all the added features, but am seeing a problem with the website/Pi where eventually the indi-allsky website hangs and is unreachable.  The Pi's are pingable, but not reachable via http/s or SSH.  I have had this happen on both a Pi3B and 4.  I just turned on debug logs, so once I have some data, I'll post it.  Just curious if anyone else has had this issue?

**Comment by aaronwmorris** [comment]
You can view the log for gunicorn at `/var/log/indi-allsky/webapp-indi-allsky.log`

Sometimes the initial request to gunicorn can take 5-15 seconds on slower systems like the Pi3.  But if there is a problem, it should show up in the log.
